examples/stocks/lib/stock_row.dart

```diff
@@ -36,28 +37,31 @@ class StockRow extends StatelessComponent {
       )
     ];
 
-    // TODO(hansmuller): An explicit |height| shouldn't be needed
-    return new Container(
-      padding: const EdgeDims(16.0, 16.0, 20.0, 16.0),
-      height: kHeight,
-      decoration: new BoxDecoration(
-        border: new Border(
-          bottom: new BorderSide(color: Theme.of(context).dividerColor)
+    return new GestureDetector(
+      onTap: onPressed,
+      child: new InkWell(
```

Comment:
Eventually we're going to want to push the gesture detector into the ink well so that it can use tap cancel to cancel the splash.